From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 20 Apr 2017 16:25:49 +0200
Subject: build: remove calling untrusted hooks

Avoid downloading untrusted executables and wasting time with a
sysroot.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
Change-Id: I616210346f3e6e1a154d05181f96060bb747206b
---
 DEPS | 31 ++-----------------------------
 1 file changed, 2 insertions(+), 29 deletions(-)

diff --git a/DEPS b/DEPS
--- a/DEPS
+++ b/DEPS
@@ -118,7 +118,7 @@ vars = {
 
   # Check out and download nacl for ChromeOS only.
   # This can be disabled e.g. with custom_vars.
-  'checkout_nacl': 'checkout_chromeos',
+  'checkout_nacl': False,
 
   # By default, do not check out src-internal. This can be overridden e.g. with
   # custom_vars.
@@ -175,7 +175,7 @@ vars = {
   # platforms, but support for other platforms may be added in the future.
   'checkout_openxr' : 'checkout_win or checkout_android',
 
-  'checkout_instrumented_libraries': 'checkout_linux and checkout_configuration != "small"',
+  'checkout_instrumented_libraries': False,
 
   # By default bot checkouts the WPR archive files only when this
   # flag is set True.
@@ -544,7 +544,6 @@ allowed_hosts = [
   'chromium-nodejs',
   'chromium-tfhub-models',
   'chrome-linux-sysroot',
-  'chromium-fonts',
   'chromium-style-perftest',
   'chromium-telemetry',
   'chromium-webrtc-resources',
@@ -2102,20 +2101,6 @@ deps = {
       'dep_type': 'cipd',
   },
 
-  'src/third_party/test_fonts': {
-      'dep_type': 'gcs',
-      'condition': 'non_git_source',
-      'bucket': 'chromium-fonts',
-      'objects': [
-          {
-              'object_name': '336e775eec536b2d785cc80eff6ac39051931286',
-              'sha256sum': 'a2ca2962daf482a8f943163541e1c73ba4b2694fabcd2510981f2db4eda493c8',
-              'size_bytes': 32624734,
-              'generation': 1647440500943755,
-          },
-      ],
-  },
-
   'src/third_party/text-fragments-polyfill/src':
     Var('chromium_git') + '/external/github.com/GoogleChromeLabs/text-fragments-polyfill.git' + '@' + 'c036420683f672d685e27415de0a5f5e85bdc23f',
 
@@ -5043,18 +5028,6 @@ hooks = [
                 '-s', 'src/build/toolchain/win/rc/mac/rc.sha1',
     ],
   },
-  {
-    'name': 'rc_linux',
-    'pattern': '.',
-    'condition': 'checkout_win and host_os == "linux"',
-    'action': [ 'python3',
-                'src/third_party/depot_tools/download_from_google_storage.py',
-                '--no_resume',
-                '--no_auth',
-                '--bucket', 'chromium-browser-clang/rc',
-                '-s', 'src/build/toolchain/win/rc/linux64/rc.sha1',
-    ]
-  },
   {
     'name': 'apache_win32',
     'pattern': '\\.sha1',
-- 

